stored_queries/ncbi_tax/ncbi_taxon_get_associated_ws_objects.yaml

```diff
@@ -48,12 +48,15 @@ query: |
       filter tax.id == @taxon_id
       filter tax.created <= @ts AND tax.expired >= @ts
       limit 1
-      for obj, e in 1..1 inbound tax ws_obj_version_has_taxon
-        filter obj.is_public or obj.workspace_id IN ws_ids
-        limit @offset, @limit
-        return {
-          ws_obj: @select_obj ? KEEP(obj, @select_obj) : obj,
-          edge: @select_edge ? KEEP(e, @select_edge) : e
-        }
+      for obj, e in 1 inbound tax ws_obj_version_has_taxon
+        for type in 1 outbound obj ws_obj_instance_of_type
+          filter obj.is_public or obj.workspace_id IN ws_ids
```

Comment:
Should this filter be in the outer loop?

Reply:
Good call, fixed